doc/release/release_0.14.rst

```diff
@@ -0,0 +1,237 @@
+Announcement: scikit-image 0.14.0
+=================================
+
+We're happy to announce the release of scikit-image v0.14.0!
+
+This is the last major release with an official support of Python 2, future
+releases will be developed broadly using Python 3 capabilities, and will be
+tested against Python 3 (or greater) environments only.
+
+This release is also a long-time support (LTS) release, which means that it will
+receive important backports and bugfixes for several years.
+
+scikit-image is an image processing toolbox for SciPy that includes algorithms
+for segmentation, geometric transformations, color space manipulation,
+analysis, filtering, morphology, feature detection, and more.
+
+For more information, examples, and documentation, please visit our website:
+
+http://scikit-image.org
+
+
+New Features
+------------
+- Manual segmentation with matplotlib:
+  ``skimage.future.manual_polygon_segmentation``,
+  ``skimage.future.manual_lasso_segmentation``. (#2584)
+- Hysteresis thresholding:
+  ``skimage.filters.apply_hysteresis_threshold``. (#2665)
+- Lookfor function: ``skimage.lookfor``. (#2713)
+- Reworked montage function: ``skimage.util.montage``. (#2626)
+- 2D and 3D segmentation with morphological snakes:
```

Comment:
- segmentation with morphological snakes:
  ``skimage.segmentation.morphological_chan_vese`` (2D),
 ``skimage.segmentation.morphological_geodesic_active_contour`` (2D and 3D). (#2791)